### PR TITLE
HEAT-200 Add slack channel link in product and team UI

### PR DESCRIPTION
**Pull request by babuharidass11**

### CODE REVIEW

File: server/views/pages/product.njk [hunk 1]
```diff
@@ -77,6 +77,9 @@
           <li><a class="govuk-link--no-visited-state" href="/drift-radiator/products/{{ product.name | toMonitorName }}">
              Deployment drift
           </a></li>
+          <li><a class="govuk-link--no-visited-state" href="slack://channel?team=T02DYEB3A&id={{ product.slackChannelId | toMonitorName }}">
```
Comment (mattops) [inline server/views/pages/product.njk:80]: do we need `toMonitorName` ?

Comment (psoleckimoj) [reply server/views/pages/product.njk:80]: No, that's a nunjucks function, no need for that here.

File: server/views/pages/team.njk [hunk 2]
```diff
@@ -37,6 +37,9 @@
           <li><a class="govuk-link--no-visited-state" href="/drift-radiator/teams/{{ team.name | toMonitorName }}">
              Deployment drift
           </a></li>
+          <li><a class="govuk-link--no-visited-state" href="slack://channel?team=T02DYEB3A&id={{ team.slackChannelId | toMonitorName }}">
+             Slack channel
+          </a></li>
```
Comment (psoleckimoj) [inline server/views/pages/team.njk:42]: Same here

File: server/views/pages/product.njk [hunk 3]
```diff
@@ -74,9 +74,14 @@
           <li><a class="govuk-link--no-visited-state" href="/monitor/product/{{  product.name | toMonitorName }}">
              Health monitor
           </a></li>
-          <li><a class="govuk-link--no-visited-state" href="/drift-radiator/products/{{ product.name | toMonitorName }}">
+          <li><a class="govuk-link--no-visited-state" href="/drift-radiator/products/{{ product.name }}">
```
Comment (psoleckimoj) [inline server/views/pages/product.njk:77]: This one needs to stay, it's just the Slack channel ID is not a monitor name so doesn't need to be piped into that function to clean it up 🙂

File: server/views/pages/team.njk [hunk 4]
```diff
@@ -34,9 +34,14 @@
           <li><a class="govuk-link--no-visited-state" href="/monitor/team/{{  team.name | toMonitorName }}">
              Health Monitor
           </a></li>
-          <li><a class="govuk-link--no-visited-state" href="/drift-radiator/teams/{{ team.name | toMonitorName }}">
+          <li><a class="govuk-link--no-visited-state" href="/drift-radiator/teams/{{ team.name }}">
```
Comment (psoleckimoj) [inline server/views/pages/team.njk:37]: Same again.

File: server/views/pages/product.njk (hunk 1 above)
Comment (babuharidass11) [reply server/views/pages/product.njk]: updated

File: server/views/pages/team.njk (hunk 2 above)
Comment (babuharidass11) [reply server/views/pages/team.njk:43]: updated

**Review by psoleckimoj — APPROVED** [review]
LGTM!

File: server/views/pages/product.njk (hunk 3 above)
Comment (babuharidass11) [reply server/views/pages/product.njk]: reverted back

File: server/views/pages/team.njk (hunk 4 above)
Comment (babuharidass11) [reply server/views/pages/team.njk]: reverted back

**Pull request merged**